db/migrate/20250512114003_move_users_identity_url_to_user_auth_provider_links.rb

```diff
@@ -52,7 +52,7 @@ def change
           slug, external_id = identity_url.split(":", 2)
           next if slug.blank? || external_id.blank?
 
-          auth_provider_id = auth_providers_data.find { |_, auth_provider_slug| auth_provider_slug == slug }.first
+          auth_provider_id = auth_providers_data.find { |_, auth_provider_slug| auth_provider_slug == slug }&.first
```

Comment:
Now re-reading this piece of code, maybe it would have been clearer if we had avoided the use of `first` from the beginning:

```suggestion
          auth_provider_id, = auth_providers_data.find { |_, auth_provider_slug| auth_provider_slug == slug }
```

WDYT? (a bit magical to just have the `,` there, but IIRC rubocop would complain about `auth_provider_id, _ =`

Reply:
I'd stay with the `first` version tbh, but, actually, fine with this one as well.